packages/reports/addon/templates/components/common-actions/schedule.hbs

```diff
@@ -81,6 +81,16 @@
           {{/power-select}}
         </fieldset>
       </div>
+      {{#if (feature-flag "enabledNotifyIfData")}}
+        <div class="schedule-modal__section-group">
+          {{x-toggle
+            size="small"
+            class="schedule-modal__toggle schedule-modal__toggle--notifyWhenEmpty"
```

Comment:
Can we rename the class name to `schedule-modal__must-have-data-toggle`? 
We don't use camel case for class names.

Reply:
The convention we have for class name is schedule-modal__{type}--{name}. I will change it to `schedule-modal__toggle--must-have-data`

Reply:
actually the convention we follow is called BEM. You can check this out for what BEM stands for.
https://css-tricks.com/bem-101/

Reply:
Oh. That's awesome. Thanks!

Reply:
`must-have-data` is not a modifier. Modifiers are things like `disabled`, `active`, `invalid`, `red`, etc.. Which is why `schedule-modal__must-have-data-toggle` is a better name according to the convention.

Reply:
I see. Let me change that!

Reply:
Just addressed the naming convention concern.